.rubocop.yml

```diff
@@ -50,6 +55,10 @@ RSpec/MultipleExpectations:
 RSpec/MultipleMemoizedHelpers:
   Enabled: false
 
+RSpec/InstanceVariable:
+  Exclude:
+    - spec/system/**/* # We use instance variables in system specs to stub Cocina operations
```

Comment:
Since we intend to skip this Rubocop violation, move it from an inline skip to a configured skip with a rationale.